src/util/mod.rs

```diff
@@ -46,6 +47,34 @@ impl DataWriter {
     }
 }
 
+pub struct BodyWriter {
+    directory: String
+}
+
+impl BodyWriter {
+    /// creates a new directory in the current working path
+    pub fn new(directory: &str) -> BodyWriter {
+        if !Path::new(directory).exists() {
+            fs::create_dir(directory).expect("Could'nt create dir.");
+        }
+
+        BodyWriter {
+            directory: directory.to_owned()
+        }
+    }
+
+    pub fn write_bodies(&self, bodies: Vec<Body> , path: String) -> std::io::Result<()> {
+        let path = format!("{}/{}.txt", self.directory, path);
+        let mut file = fs::File::create(path)?;
+        for body in bodies {
+            write!(file, "{}-{}-{}-{}-{}\n", body.mass, body.position.x, body.position.y,
```

Comment:
Formatting the body data with dashes (`-`) is a bit odd and makes it hard to read. I would suggest to separate values with a comma (`,`) like we do when writing points.